src/Month.js

```diff
@@ -12,7 +12,7 @@ import getPosition from 'dom-helpers/query/position'
 import raf from 'dom-helpers/util/requestAnimationFrame'
 
 import Popup from './Popup'
-import Overlay from 'react-overlays/lib/Overlay'
+import { Overlay } from 'react-overlays'
```

Comment:
lets keep the cherry picked import i don't want to rely on treeshaking working correctly if we don't need to

Reply:
`import Overlay from 'react-overlays/Overlay'`

Reply:
Done